pilot/pkg/networking/core/v1alpha3/fake.go

```diff
@@ -208,6 +208,9 @@ func (f *ConfigGenTest) SetupProxy(p *model.Proxy) *model.Proxy {
 	if p.ConfigNamespace == "" {
 		p.ConfigNamespace = "default"
 	}
+	if p.Metadata.Namespace == "" {
```

Comment:
Is this related to the fix?

Reply:
Yes, authn code looks for this and its not set in the test. This is just defaulting for test.

Maybe in the future we should make all code consistently use only ConfigNamespace and not Metadata.Namespace